core/arch/arm/plat-stm32mp1/shared_resources.c

```diff
@@ -692,8 +692,12 @@ static void check_rcc_secure_configuration(void)
 		}
 	}
 
-	if (have_error)
-		panic();
+	if (have_error) {
+		if (IS_ENABLED(CFG_WARN_INSECURE))
+			EMSG("Warning: CFG_WARN_INSECURE allows insecure RCC configuration");
```

Comment:
Since `CFG_WARN_INSECURE` can now change the behavior as opposed to just showing a warning, I suggest it should be renamed `CFG_INSECURE` and documented properly.

Reply:
Good point. Note that we also have changed behavior in `core/tee/tee_ree_fs.c` concerning the non-volatile counter.

Reply:
Do you want I create the change?

Reply:
Yes, one commit to rename the CFG first would be good (mentioning the volatile counter change as a reason to rename, and others to come).

Reply:
ok, i'll post in this P-R.